xarray/tests/test_dask.py

```diff
@@ -1129,7 +1129,8 @@ def test_unify_chunks(map_ds):
 def test_unify_chunks_shallow_copy(obj, transform):
     obj = transform(obj)
     unified = obj.unify_chunks()
-    assert_identical(obj, unified) and obj is not obj.unify_chunks()
```

Comment:
Introduced in #3276 / 587963c0683891013e298e56729aefb94810a1e408548bbe4488c42ba403c892. Can't understand why `and` was expected to do.

Reply:
How about commenting it out with a `#` for now?